LibVLCSharp.Forms/Shared/Converters/BufferingProgressToBoolConverter.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Globalization;
+using Xamarin.Forms;
+
+namespace LibVLCSharp.Forms.Shared.Converters
+{
+    /// <summary>
+    /// Converts a value not equals to 0 and 1 to true.
+    /// </summary>
+    public class BufferingProgressToBoolConverter : IValueConverter
+    {
+        /// <summary>
+        /// Modifies the source data before passing it to the target for display in the UI.
+        /// </summary>
+        /// <param name="value">The source data being passed to the target.</param>
+        /// <param name="targetType">The type of the target property.</param>
+        /// <param name="parameter">An optional parameter to be used in the converter logic.</param>
+        /// <param name="culture">The culture of the conversion.</param>
+        /// <returns>true if value is not equals to 0, false otherwise</returns>
```

Comment:
OK, it returns true if the value is not equals to 0 and 1, false otherwise. The goal is to hide the progress bar when the buffering progress value is equal to 0 or 1 (so to show it when the progress value is beetween 0 and 1).